addons/mail/models/mail_channel.py

```diff
@@ -256,6 +256,18 @@ def write(self, vals):
                 ('res_id', 'in', self.ids)
             ])._moderate_accept()
 
+        # Reflect changes when switching a channel to a mailing one or vice versa
+        if 'email_send' in vals:
+            notifications = []
+            for channel in self:
+                notifications.append([(self._cr.dbname, 'mail.channel', channel.id), {
+                    'info': 'channel_updated',
+                    'channel': {
+                        'mass_mailing': channel.email_send,
+                    },
+                }])
+            self.env['bus.bus'].sendmany(notifications)
+
```

Comment:
Since this is master, let's just do it correctly immediately. So:
- This should not be limited to `email_send` anymore.
- This should send the full `channel_info()` result whenever anything changed.
- `channel_info()` should be changed to never send user-specific info.
`_broadcast` solves it by sending the info to each user, but we probably want to split the two concerns.
`channel_info` (or a new method to avoid confusion) for pure channel specific stuff, and another method for user specific channel info (the content of `channel.partner` basically). This way we don't need to compute and send a huge amount of user specific stuff when we change the channel config (or vice-versa).

Reply:
Please no. Don't generate queries and bus notifications if I change something else than email_send ... Or trigger a refresh like every other write in Odoo.